src/main/java/org/railway/BasePage.java

```diff
@@ -36,14 +42,33 @@ public static void switchToNewTab() {
         Driver.driver.switchTo().newWindow(WindowType.TAB);
     }
 
-    public static void navigateToRailway() {
-        String railwayUrl = ConfigDriver.getProperty("railway.url");
-        Driver.driver.get(railwayUrl);
-        railway = Driver.driver.getWindowHandle();
-    }
-
     public void clickTab(String tabname) {
         By tabName = By.xpath(String.format(tab, tabname));
-        Action.click(Action.find(tabName));
+        Action.click(tabName);
+    }
+
+    public boolean checkTitle(String message) {
```

Comment:
these methods return true/false, you should rename the method with prefix 'is/does,...' (isTitleDisplayed, doesControlExist,...)